From 53e83adf2944730b87e48cc39b26b9d23b4c3aad Mon Sep 17 00:00:00 2001
From: Salvo Giangreco <giangrecosalvo9@gmail.com>
Date: Fri, 23 May 2025 17:01:56 +0200
Subject: [PATCH] Hide call blocking card

Signed-off-by: Ksawlii <ksawery.blaszczak@proton.me>
---
 smali_classes2/yd/j.smali | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/smali_classes2/yd/j.smali b/smali_classes2/yd/j.smali
index af02d02c..aab8bb7e 100644
--- a/smali_classes2/yd/j.smali
+++ b/smali_classes2/yd/j.smali
@@ -112,7 +112,7 @@
 .method public B()Z
     .locals 0
 
-    const/4 p0, 0x1
+    const/4 p0, 0x0
 
     return p0
 .end method
-- 
2.49.0

